### DISCUSSION TITLE
Self signed certificate in chain when making a call to jira

### DISCUSSION THREAD

**Discussion by voytec86** [actions/github-script#483] [Q&A]
Hi,
On GHES, using github-script action I'm making calls to corporate jira, and no mather what I do I get " self signed certificate in chain" error.
actions checkout is working fine so connection to GHES works.
I've tried this solution https://github.com/actions/checkout/issues/362#issuecomment-705962120 but instead of GEHS pointed to jira, but still no whatever I do node js doesn't see certificate.
I've tried this also: https://stackoverflow.com/questions/45088006/nodejs-error-self-signed-certificate-in-certificate-chain
but still.
Only this that allows to pass this call is setting:
NODE_TLS_REJECT_UNAUTHORIZED=0 
in .env file, but it feels like leaving dors open just because lock isn't working.

What I'm missing?

**Comment by hariudkmr** [comment]
Thankyou so much your suggestion helped to resolve the issue
NODE_TLS_REJECT_UNAUTHORIZED=0
